src/main/java/com/dougnoel/sentinel/steps/ImageSteps.java

```diff
@@ -27,22 +27,23 @@ public static void before(Scenario scenario) {
 	/**
 	 * Takes a screenshot of the given element and stores it for later comparison
 	 * @param elementName String name of the element to screenshot
-	 * @throws IOException
+	 * @throws IOException If the expected image to be created fails to save
 	 */
-    @When("^I take a screenshot of (?:the|a) (page|.*)$")
```

Comment:
I feel like the term page here is ambiguous with page object, but I can't think of a better term.

Reply:
Problem is a user using windows writing tests. We don't necessarily anticipate the person writing the steps to be writing the tests. So it just made sense to allow page or window.

I don't really know many people who would call explorer.exe a page even if they were using a page object.